CUETools.Codecs.FlaCuda/FlaCudaWriter.cs

```diff
@@ -1880,11 +1880,11 @@ int write_headers()
 			return header_size;
 		}
 
-		int flake_encode_init()
+		int Flake_encode_init()
 		{
 			int i, header_len;
 
-			//if(flake_validate_params(s) < 0)
+			//if(Flake_validate_params(s) < 0)

```

Comment:
Please undo. The lower case letter at the beginning of the method name should stay, like in other methods.
Furthermore `flake_validate_params` has been commented out throughout the project and is not relevant to this issue.